src/Tribe/Views/V2/Views/Widgets/List_Widget_View.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * The List Widget View.
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Views\Widgets;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
+use Tribe__Context;
+use Tribe__Utils__Array as Arr;
+
+class List_View extends View {
+	use List_Behavior;
```

Comment:
:no_entry_sign: Missing doc comment for class List_View

<sub>hash: 03b3b610211151cbfdebeb3f5e34717e</sub>